py3-setuptools.yaml

```diff
@@ -29,13 +29,13 @@ environment:
       - busybox
       - ca-certificates-bundle
       - build-base
-      - python3
+      - system-python
       - py3-appdirs
       - py3-more-itertools
       - py3-ordered-set
       - py3-packaging
       - py3-parsing
-      - py3-setuptools
+      - py3-setuptools-stage0
```

Comment:
Did this otherwise there are package conflicts during the CI build. The already built `py3-setuptools` depends on `python3`, which overwrites files provided by `system-python` and makes apk unhappy.